src/main/java/com/minecolonies/coremod/commands/ActionArgumentType.java

```diff
@@ -232,7 +236,8 @@ public int allowedSpaceCount()
     }
 
     @NotNull
-    private List<String> getCitizenTabCompletions(@NotNull final ActionMenuState actionMenuState, final String potentialArgumentValue)
+    private List<String> getCitizenTabCompletions(@NotNull final MineColonyDataProvider mineColonyDataProvider,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 29 to the 20 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)